sdk/communication/azure-communication-identity/azure/communication/identity/_shared/models.py

```diff
@@ -103,11 +103,11 @@ def __eq__(self, other):
 
 def _communication_user_raw_id(identifier: CommunicationUserIdentifier) -> str:
     if identifier.raw_id:
-        return identifier.raw_id
-    return identifier.properties["id"]
+        return str(identifier.raw_id)
+    return str(identifier.properties["id"])
```

Comment:
Is this to fix a mypy error or was this function not returning str previously?

Reply:
This is to fix mypy error